oap-server/server-configuration/configuration-api/src/main/java/org/apache/skywalking/oap/server/configuration/api/ConfigChangeWatcher.java

```diff
@@ -60,14 +62,25 @@ public String toString() {
     public static class ConfigChangeEvent {
         private String newValue;
         private EventType eventType;
+        private String groupItemName;
 
         public ConfigChangeEvent(String newValue, EventType eventType) {
             this.newValue = newValue;
             this.eventType = eventType;
         }
+
+        public ConfigChangeEvent(String groupItemName, String newValue, EventType eventType) {
+            this.groupItemName = groupItemName;
+            this.newValue = newValue;
+            this.eventType = eventType;
+        }
     }
 
     public enum EventType {
         ADD, MODIFY, DELETE
     }
+
+    public enum WatchType {
+        SIMPLE, GROUP
```

Comment:
```suggestion
        SINGLE, GROUP
```

I think SINGLE is a better word?